latex/Research_Strategy.tex

```diff
@@ -243,29 +244,32 @@
 % FIGURE: TIMELINE
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 \begin{figure}[h]
+\vspace{-0.10in}
 \begin{centering}
 %\resizebox{!}{3.2in}{\includegraphics{figures/Timeline.pdf}}
 \includegraphics[width=\textwidth]{figures/Timeline2_cropped.pdf}
 \end{centering}
-\vspace{0.1in}
+\vspace{-0.25in}
 \caption{\footnotesize {\bf Timeline for our activities.} Activities covered by this grant include data collection and SAMPL challenges on our three major components (physical properties, host-guest binding, and protein-ligand binding), with each challenge cycle color-coded separately.  
 Data collection within each Aim is shown by a colored bar indicating what is measured and curated.
-Data collection/curation is followed by a submission window for that challenge component, then all results and analysis are returned to participants and posted on the SAMPL website; this also will nucleate more detailed discussion on the relevant Slack channel (which is expected to already be underway as participants prepare during the data collection phase). 
+Data collection/curation is followed by a submission window for that challenge component, then all results and analysis are returned to participants and posted on the SAMPL website; this also will nucleate more detailed long-term discussion on the relevant Slack channel.
 Each component will then wrap up with a focused virtual meeting where participants focus on lessons learned and areas which need further exploration; these will be recorded and posted on our website to assist in rapid dissemination of new insights. 
 Virtual meetings precede the submission window for the next SAMPL challenge, giving the opportunity to incorporate lessons learned for the next challenge.
+Release of curated datasets as high-quality benchmark sets will immediately follow data unblinding.
```

Comment:
I'll tweak this slightly to something more concise, perhaps "Immediately following the submission window, curated datasets will be released as high quality benchmarks."